api/namespace.go

```diff
@@ -0,0 +1,131 @@
+package api
+
+import (
+	"fmt"
+	"time"
+)
+
+// Namespace is the configuration of a single namespace. Namespacing is a Consul Enterprise feature.
+type Namespace struct {
+	// Name is the name of the Namespace. It must be unique and
+	// must be a DNS hostname. There are also other reserved names
+	// that may not be used.
+	Name string `json:"Name,omitempty"`
```

Comment:
Is it possible for `Name` to actually be empty?

Reply:
Our API functions do not allow it. Although the namespace API can technically fill in the Namespace for you from whats provided in the URL.

I went ahead and removed that part of the tag.